### PR TITLE
Feature cta card

### PR DESCRIPTION
**Pull request by abouman76**
# What this pull-request does.

- Made CTA card component, with button component.
- Added styling with css
- Installed icon library (run npm install after this pull rquest)

Schermafbeelding 2021-07-07 om 19 25 35

### CODE REVIEW

File: src/components/CallToAction.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+} */
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;500&display=swap");
+
+.main-cta-wrapper {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  width: 360px;
```
Comment (sjaak1992) [inline src/components/CallToAction.css:12]: Maybe we should implement rem values to make it also more responsive

**Review by sjaak1992 — COMMENTED** [review]
Good job! <3_<3 I was only wandering if we should implement rem values, from the beginning on so we can make it responsive

File: src/components/CallToAction.css (hunk 1 above)
Comment (Reinoptland) [reply src/components/CallToAction.css:12]: I think this a great idea 👍 

We can setup a font-size on the html with media queries and start specificying sizes in rems @sjaak1992 

(In the case of the width of this card, it's 360px on the desktop and 360px on the mobile design, so maybe in this case the px's are okay)

**Comment by abouman76** [comment]
Good idea to switch to rem sizes. 

Verstuurd vanaf mijn iPhone

File: src/components/CallToAction.css (hunk 1 above)
Comment (sjaak1992) [reply src/components/CallToAction.css:12]: Yes perfect!

File: src/components/BtnCta.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+.btn {
+  height: 48px;
+  font-weight: bold;
+  outline: none;
+  border: none;
+  border-radius: 6px;
+  justify-content: center;
+  padding: 0 10px;
+  width: 240px;
+  font-family: "Roboto Mono", monospace;
```
Comment (Reinoptland) [inline src/components/BtnCta.css:12]: Let's make a central variable for fonts as well

File: src/components/BtnCta.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+.btn--purple {
+  background-color: #730ec3;
```
Comment (Reinoptland) [inline src/components/BtnCta.css:16]: And variables for colors

File: src/components/BtnCtA.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./BtnCta.css";
+
+const Button = ({ variant, text }) => {
+  return (
+    <button className={`btn btn--cta btn--${variant}`} type="button">
+      {text}
```
Comment (Reinoptland) [inline src/components/BtnCtA.js:7]: Another way you can display content inside a React component is to use `props.children`

We could do something like so: 

Define like this:
```js
const Button = (props) => {
  return (
    <button>{props.children}<button>
  )
}
```

Use like this: 

```
<Button>
  <SomeIconOnTheButton />
  Some text on the button
</Button>
```

This makes it easier to reuse the button easily

**Review by Reinoptland — APPROVED** [review]

**Pull request merged**